json_rpc/rpcproxy.nim

```diff
@@ -0,0 +1,38 @@
+import servers/[socketserver, httpserver]
```

Comment:
We typically write `import` once and then newline, and differentiating stdlib imports, module imports and local imports with again new lines. Local imports should use relative paths also.

See also: https://status-im.github.io/nim-style-guide/language.import.html